tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -46,6 +46,7 @@
 
 import rosgraph.names
 import rospkg
+import loader
```

Comment:
Aren't relative imports of the folloing form better?

    from .loader import convert_value